playbooks/roles/backups/files/backup.py

```diff
@@ -37,6 +35,8 @@ def upload_to_s3(file_path, bucket, aws_access_key_id, aws_secret_access_key):
         aws_secret_access_key: An AWS secret access key.
     """
 
+    from filechunkio import FileChunkIO
```

Comment:
@johnbaldwin `filechunkio` is only needed for uploading to S3 buckets, so the [Ansible task](https://github.com/appsembler/configuration/blob/6185813d62a53e2adff4846f5c55ea3effecc615/playbooks/roles/backups/tasks/main.yml#L24-L27) only installs the package if we're using S3. Does this seem like a reasonable thing to do?